src/model.rs

```diff
@@ -209,16 +226,248 @@ impl QueryResponse {
         }
     }
 
+    pub fn get_affected_rows(&self) -> u32 {
```

Comment:
I think this can just be a getter by using `#[pyo3(get)]` on the field in struct definition?

Reply:
Thanks, done.

Reply:
I thought this had been fixed... @Rachelint Maybe you can create another PR to push the commit about this refactor.